sdk/communication/azure-communication-callautomation/tests/test_callautomation_client.py

```diff
@@ -64,13 +63,11 @@ def mock_send(request, **kwargs):
 
         call_invite = CallInvite(
             target=user,
-            voip_headers={"foo": "bar"},
             source_display_name="WRONG"
         )
         call_connection_properties = call_automation_client.create_call(
             target_participant=call_invite,
             callback_url=self.callback_url,
-            voip_headers={"foo": "bar"},
```

Comment:
I believe we just removed voip_headers and sip_headers from CallInvite. At other places we can keep it.